apps/server/src/modules/learnroom/controller/course.controller.ts

```diff
@@ -131,7 +131,7 @@ export class CourseController {
 		};
 	}
 
-	@Get(':courseId')
+	@Get(':courseId/cc-metadata')
```

Comment:
change method name to "getCourseCcMetadataById"

Reply:
done